src/coreclr/nativeaot/Runtime/RhConfig.h

```diff
@@ -11,6 +11,8 @@
 // Values can also be embedded in the compiled binary.
 //
 
+#ifndef __RHCONFIG_H__
+#define __RHCONFIG_H__
```

Comment:
nit: by convention, __ prefixed / suffixed macros are reserved for system headers.
```suggestion
#define RHCONFIG_H
```